CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 3.0.0-beta.7
+
+- Update minim to 0.20.5.
+
```

Comment:
Minim-parse-result?

Reply:
I don't think it's worth mentioning as the only change in 0.10.1 is to JSDoc. This brings no functional change to fury.js.